src/screens/Resources/variables.js

```diff
@@ -0,0 +1,11 @@
+export const collections = {
```

Comment:
We can probably rename this file to `constants` and move it up so other components can use it.